mantidproject/mantid
# LoadEventNexus duplicate code

While helping out people loading data from LENS, I discovered that `LoadEventNexus` had an independent implementation of loading event monitors which gave different results from `LoadNexusMonitors`. Talking with developers at SNS, we decided to change the parameter names to clarify whether it was event or histogram monitors that were being loaded. The result is having `LoadNexusMonitors` always used from `LoadEventNexus` and documentation cleanup. The tests that were removed were verifying that broken behavior continued. There is much better coverage in `LoadNexusMonitors` already so the likelihood of new bugs being introduced should actually be lower.

**To test:**

Try loading your favorite nexus files with monitors and see if things still work. There were some CORELLI files that had to be tried with a particular (and strange) set of parameters in `LoadEventNexus`. Ticking the `LoadMonitors` option should be enough now.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
